src/actions/themeActions.ts

```diff
@@ -1,13 +1,23 @@
 import {Constants} from './types';
 
-interface ChangeTheme {
-  type: Constants.CHANGE_THEME;
-  isDark: boolean;
+interface SetLightTheme {
+  type: Constants.SET_LIGHT_THEME;
+  theme: string;
 }
 
-export const changeTheme = (isDark: boolean): ChangeTheme => ({
-  type: Constants.CHANGE_THEME,
-  isDark: isDark,
+export const setLightTheme = (theme: string): SetLightTheme => ({
```

Comment:
Can we implement setTheme action where we will receive theme enum - string as a result theme that we need to setup....